warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctions.java

```diff
@@ -61,9 +65,9 @@ protected static String getHitTermString(Object valueTuple) {
     }
     
     public static Collection<?> between(Object field, String left, boolean leftInclusive, String right, boolean rightInclusive) {
-        // we only want to test against the normalized variant to be consistent with the DatawaveArithmetic
-        String fieldValue = ValueTuple.getNormalizedStringValue(field);
-        if (fieldValue != null) {
+        if (field != null) {
+            // we only want to test against the normalized variant to be consistent with the DatawaveArithmetic
+            String fieldValue = ValueTuple.getNormalizedStringValue(field);
```

Comment:
Isn't the null check for `fieldValue` still needed, since ValueTuple.getNormalizedStringValue can return null?

Reply:
can't hurt

Reply:
I didn't look like ValueTuple.getNormalizedStringValue should return a null unless it was passed a null, but I added two additional null checks for safety and readability.

Also retargetted the PR to release/version3.2